src/workers/notifications/utils.ts

```diff
@@ -151,10 +201,16 @@ export async function articleNewCommentHandler(
       : NotificationType.ArticleNewComment;
 
   if (source.type === SourceType.Squad) {
-    const members = await getSubscribedMembers(con, type, post.id, {
-      userId: In(users),
-      sourceId: source.id,
-      role: Not(SourceMemberRoles.Blocked),
+    const members = await getOptOutMembers({
```

Comment:
I'm a bit confused by the naming or the function maybe?
you get opted out members, but those get the email?